xarray/tests/test_sparse.py

```diff
@@ -719,13 +720,17 @@ def test_sparse_dask_dataset_repr(self):
         ds = xr.Dataset(
             data_vars={"a": ("x", sparse.COO.from_numpy(np.ones(4)))}
         ).chunk()
+        if Version(sparse.__version__) >= Version("0.16.0"):
+            meta = "sparse.numba_backend._coo.core.COO"
```

Comment:
This seems to be because dask extracts the `__module__` which has now moved I guess.

https://github.com/dask/dask/blob/8d53292b05535c41b478ec882dd65d485bf1eb38/dask/array/core.py#L1632-L1633

cc @hameerabbasi and @fjetter

Reply:
Would there be a way to avoid depending on `__module__`? Alternatively, would there be a preference about what to keep it constant at? I'm happy to make a post1 release fixing this.

Reply:
Ah, I see. It's about keeping the Dask repr constant, which in turn uses `__module__`. I could change the module of all our functions/methods to be just `"sparse"` and making a post0 release.